arm-apimanagement/2015-09-15/swagger/service.json

```diff
@@ -0,0 +1,1288 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "APIManagementServiceClient",
+    "description": "REST API for Azure APIManagement Service",
+    "version": "2015-09-15"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_GetApiManagementServiceOperation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
```

Comment:
please make sure that the description is not empty or completely absent. description forms the documentation of the client library, MSDN REST API doc (in the near future) and also the help for (autogenerated) CLI/powershell commands(in the near future).